### ISSUE TITLE
A markdown kernel?

### ISSUE THREAD

**Issue by BoPeng** [vatlab/sos-notebook#57]
Currently we have a `%render` magic to process whatever output from a cell in markdown or other formats. This is a powerful mechanism but it is complicated if you just want to enter some markdown.

A markdown kernel is a bad idea because there are already markdown cells that are handled by Jupyter, and there is no concept of variable etc in markdown. However, with SoS, a markdown kernel can be useful  with something like

```
%expand
* This is result whatever {result}
```

Basically, the `%expand` magic would make the kernel useful in an SoS environment.

**Comment by BoPeng** [comment]
Implemented in [markdown-kernel](https://github.com/vatlab/markdown-kernel)
